src/data/gateway-data-source.ts

```diff
@@ -86,6 +87,8 @@ export class GatewayDataSource implements ContiguousDataSource {
       responseType: 'stream',
     });
 
+    metrics.arweaveTxFetchCounter.inc({ node_type: 'trusted' });
```

Comment:
This one should be removed since it's a data rather than header request.

Reply:
Done